actions/changelog-generate/action.yaml

```diff
@@ -21,7 +21,7 @@ inputs:
   config:
     description: Path to the changelog config JSON file
     required: false
-    default: "$GITHUB_ACTION_PATH/changelog-config.json"
+    default: "default"
```

Comment:
Why not use an empty string as the default here? That would also solve the issue with having to set defaults in every workflow that uses the action

Reply:
good plan

Reply:
changed